help/md/median.md

```diff
@@ -1,9 +1,23 @@
 ## name
 median
 ## title
+Median of a set of numbers
 ## description
+Finds the median of a sorted vector of numbers.
 ## details
+The vector is sorted when `median` is used finding the
+number of the sorted values with an equal amount of numbers that
+are greater than or less than that value. If the length of the 
+vector is even there will be no two such values, in that case
+the two are averaged arithmetically.
```

Comment:
I'm having trouble parsing the 1st sentence -- are there missing or extra words? For the 2nd sentence, I'd propose:

"If the length of the vector is even, there will be no such value. In that case, the two are averaged automatically."